### PR TITLE
Add a blog post talking about the development focus areas for the next year

### PR DESCRIPTION
**Pull request by rjmccall**

### CODE REVIEW

**Comment by rjmccall** [comment]
@swift-ci Please test

**Comment by tkremenek** [comment]
I would like to be the one who merges this PR, once the Website WG reviews.  Thanks.

**Review by tomerd — APPROVED** [review]

**Comment by tkremenek** [comment]
@swift-ci test

File: _posts/2022-11-17-focus-areas-next-year.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+  - Developing features to give programmers explicit control over the ownership of values in memory. This includes features to prohibit implicit copies, transfer ownership between contexts, and explicitly “borrow” values without copying them.
+
+  - Adding basic support for non-copyable types. This will provide new ways to achieve high performance by restricting the lifecycle of critical values. These controls will in turn enable new ways to work with data in memory, combining the performance of current “unsafe” constructs with the safety of Swift’s standard library features.
+
+* To further empower the creation of rich libraries and DSLs, developing the basics of a [procedural macro system](https://forums.swift.org/t/a-possible-vision-for-macros-in-swift/60900).  This effort will start with the creation of a vision document that will lay out a design for what macros can achieve in Swift and how they might fit into the language.
+
+* Introducing support for [C++ interoperability](https://forums.swift.org/t/report-swift-and-c-interoperability-project-progress-in-the-swift-5-7-time-frame/61005) under the auspices of the new dedicate C++ Interoperability Workgroup:
```
Comment (hyp) [inline _posts/2022-11-17-focus-areas-next-year.md:40]: Should `dedicate` here be `dedicated` or am I wrong? Isn't dedicate is a verb?

File: _posts/2022-11-17-focus-areas-next-year.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+## Summary
+
+That's it for now!  We've tried to cover the most important work happening across the project, but there are a lot of smaller things going on, too, that we just didn't have space to cover.  We hope that putting this information together in one place like this has been useful.
+
+If you're interested in getting involved, there are a lot of ways to do it.  Language changes will be going through the normal evolution process, so keep an eye out for [discussion threads, pitches, and proposal reviews](https://forums.swift.org/c/evolution/18) about the work that you're interested in.  If you have questions or feedback about any of the items in this post, you can reach out to the appropriate workgroup, or, if you're not sure what that is, on the forums thread associated with this post (<link here>).
```
Comment (alexandersandberg) [inline _posts/2022-11-17-focus-areas-next-year.md:144]: >...on the forums thread associated with this post (<link here>).

Just leaving this comment here so we don't forget to update this link placeholder. 🙂

**Review by alexandersandberg — APPROVED** [review]
🎉

File: _posts/2022-11-17-focus-areas-next-year.md (hunk 1 above)
Comment (airspeedswift) [reply _posts/2022-11-17-focus-areas-next-year.md:40]:
```suggestion
* Introducing support for [C++ interoperability](https://forums.swift.org/t/report-swift-and-c-interoperability-project-progress-in-the-swift-5-7-time-frame/61005) under the auspices of the new dedicated C++ Interoperability Workgroup:
```

File: _posts/2022-11-17-focus-areas-next-year.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+## Package registry
+
+Development on the Swift package manager is focused on starting work on an open source package registry server implementation in concert with the community. The goal is to create the technical components required to transition Swift packages ecosystem from source control based dependencies to registry based ones, enhancing the security and reliability of the ecosystem.
+
+## Implementation improvements
+
+In addition to the feature work above, compiler developers are focused on several improvements at the implementation level:
+
+- Developing a Swift parser that's written in pure Swift and feature-complete with the current C++ implementation.  This opens up a lot of new directions in tooling development, and someday it may even be able to supplant the C++ parser.
```
Comment (airspeedswift) [inline _posts/2022-11-17-focus-areas-next-year.md:70]:
```suggestion
- Developing a Swift parser that's written in pure Swift and feature-complete with the current C++ implementation.  This opens up a lot of new directions in tooling development and, once at feature parity, will supplant the C++ parser.
```

"Some day may even be able" seems way over hedged. It _will_ replace the C++ parser.

**Comment by tkremenek** [comment]
@swift-ci test

**Comment by tkremenek** [comment]
@swift-ci test

**Comment by rjmccall** [comment]
@swift-ci Please test

**Comment by tkremenek** [comment]
@swift-ci test

**Review by shahmishal — APPROVED** [review]

**Comment by cthielen** [comment]
@swift-ci test

**Review by digimarktech — APPROVED** [review]
Love it!

**Pull request merged**